compiler/rustc_data_structures/src/lib.rs

```diff
@@ -43,6 +43,7 @@ use std::fmt;
 pub use atomic_ref::AtomicRef;
 pub use ena::{snapshot_vec, undo_log, unify};
 pub use rustc_index::static_assert_size;
+pub use {either, indexmap, smallvec, thin_vec};
```

Comment:
```suggestion
// Re-export some data-structure crates
pub use {either, indexmap, smallvec, thin_vec};
```
or something like the comment on tracing?